apis/v1beta1/common.go

```diff
@@ -186,6 +186,9 @@ type OpenTelemetryCommonFields struct {
 	// HostNetwork indicates if the pod should run in the host networking namespace.
 	// +optional
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// HostPID indicates if the pod should have access to the host process ID namespace.
+	// +optional
+	HostPID bool `json:"hostPID,omitempty"`
```

Comment:
because you add this in common.go, I guess this was also added to target-allocator. Please remove and regenerate.